app/src/main/java/vn/edu/usth/minigh/api/Interface.kt

```diff
@@ -24,6 +28,21 @@ interface GitHub {
     @GET("/repos/{repoName}/readme")
     suspend fun readme(@Path("repoName", encoded=true) repoName: String): Readme
 
+    @GET("/search/issues")
+    suspend fun issueUser(@Query("q") author:String): IssueOfUser
+
```

Comment:
Fyi there is [issues of authenticated user](https://docs.github.com/en/rest/reference/issues#list-user-account-issues-assigned-to-the-authenticated-user). But well if this works then it doesn't matter.